src/Page/Auth.elm

```diff
@@ -0,0 +1,698 @@
+                            p [ class "text-center" ]
+                                [ text "Vous avez maintenant accès au détail des impacts, à utiliser conformément aux "
+                                , a [ href Env.gitbookUrl ] [ text "conditions" ]
+                                , text "."
+                                ]
+
+                          else
+                            text ""
+                        , viewAccount model
+                        , button
+                            [ onClick Logout
+                            , class "btn btn-link mb-3"
+                            ]
+                            [ text "Déconnexion" ]
+                        ]
+
+                  else
+                    div [ class "row d-flex justify-content-center" ]
+                        [ div [ class "col-xl-12" ]
+                            [ Alert.simple
+                                { level = Alert.Info
+                                , close = Nothing
+                                , title = Nothing
+                                , content =
+                                    [ div [ class "fs-7" ]
+                                        [ """Pour avoir accès au détail des impacts, il est nécessaire de s'enregistrer et
+                                    valider que vous êtes Français, et que vous n'utiliserez pas ces données à des fins
+                                    commerciales."""
+                                            |> Markdown.simple []
+                                        ]
+                                    ]
+                                }
+                            ]
+                        , viewLoginRegisterForm model
+                        ]
+                ]
+            ]
+      ]
+    )
+
+
+viewAccount : Model -> Html Msg
+viewAccount model =
+    div [ class "card shadow-sm col-sm-6" ]
+        [ dl []
+            [ dt []
+                [ text "Email : " ]
+            , dd []
+                [ text model.user.email ]
+            , dt []
+                [ text "Nom : " ]
+            , dd []
+                [ text model.user.lastname ]
+            , dt []
+                [ text "Prénom : " ]
+            , dd []
+                [ text model.user.firstname ]
+            , dt []
+                [ text "Organisation : " ]
+            , dd []
+                [ text model.user.company ]
+            , dt []
+                [ text "TOKEN : " ]
+            , dd []
+                [ text model.user.token ]
+            ]
+        ]
+
+
+viewLoginRegisterForm : Model -> Html Msg
+viewLoginRegisterForm model =
+    div [ class "card shadow-sm col-sm-6" ]
+        [ div [ class "card-header px-0 pb-0 border-bottom-0" ]
+            [ ul [ class "Tabs nav nav-tabs nav-fill justify-content-end gap-2 px-2" ]
+                ([ ( "Inscription", Register )
+                 , ( "Connexion", Authenticate )
+                 ]
```

Comment:
These tabs look like this:

![image](https://github.com/MTES-MCT/ecobalyse/assets/41547/7fe6a48b-43ee-41ed-9141-25d7935222a3)

While they should look like this:

![image](https://github.com/MTES-MCT/ecobalyse/assets/41547/3162b657-02c1-4c22-a950-76c7e016b858)

Also a bit more top margin would be nice.

Reply:
Side note, if you accept I can probably push a few UI enhancements on the branch.

Reply:
Would greatly appreciate that, thanks a lot 🙏

Reply:
Done in [Improve registration form styling.](https://github.com/MTES-MCT/ecobalyse/pull/565/commits/8d8e205c687bc8f51af1c8603866020bb5441851)